python/ml4ir/base/model/scoring/interaction_model.py

```diff
@@ -39,8 +39,8 @@ class UnivariateInteractionModel(InteractionModel):
     def __init__(
         self,
         feature_config: FeatureConfig,
-        feature_layer_keys_to_fns: dict,
         tfrecord_type: str,
```

Comment:
A naive ask here: what is exactly an Interaction model? why  do we call it like this? Is it a family of feedforward NNs or something in these lines? I have seen it for ranking and ep but I do not grasp the term well

Reply:
We can work on the term. But it is the first layer of the model where you define the feature functions for each feature and then combine them into a dense layer.

Use this layer to define embeddings, LSTMs, etc.